kahuna/public/js/search/query.js

```diff
@@ -23,7 +23,7 @@ query.controller('SearchQueryCtrl',
                  ['$scope', '$state', '$stateParams', 'onValChange', 'mediaApi', 'track',
                  function($scope, $state, $stateParams, onValChange , mediaApi, track) {
 
-    var ctrl = this;
```

Comment:
`const` **doesn't** enforce immutability, only that the variable isn't reassigned. (It's the same as `val` in Scala, vs `var`, btw. Only that most types in Scala are immutable)